src/textual/geometry.py

```diff
@@ -307,21 +347,29 @@ def contains_region(self, other: Region) -> bool:
         )
 
     def translate(self, x: int = 0, y: int = 0) -> Region:
-        """Move the origin of the Region.
+        """Create a new instance of the same size with the origin shifted by `(x, y)`.
 
         Args:
-            translate_x (int): Value to add to x coordinate.
-            translate_y (int): Value to add to y coordinate.
+            x (int): Value to add to x-coordinate for the new region. Optional; defaults to 0.
+            y (int): Value to add to y-coordinate for the new region. Optional; defaults to 0.
 
         Returns:
-            Region: A new region shifted by x, y
+            Region: A new `Region` object of the same size, shifted by `(x, y)`.
         """
 
         self_x, self_y, width, height = self
-        return Region(self_x + x, self_y + y, width, height)
+        return Region((self_x + x), (self_y + y), width, height)
 
     def __contains__(self, other: Any) -> bool:
-        """Check if a point is in this region."""
+        """Check if a point is in this region.
+        
+        Returns:
+            If `other` is of type `Region` or `tuple[(int | float), (int| float)]`:
```

Comment:
This doesn't read like English, we probably don't need the types. Why the colon, is it intended to be psuedo-code?